recharts-bar-chart/src/App.js

```diff
@@ -0,0 +1,145 @@
+import {
+  client,
+  useConfig,
+  useElementData,
+  useElementColumns,
+} from "@sigmacomputing/plugin";
+import "./App.css";
+import React from "react";
+import {
+  ResponsiveContainer,
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+} from "recharts";
+
+client.config.configureEditorPanel([
+  { name: "source", type: "element" },
+  {
+    name: "dimension",
+    type: "column",
+    source: "source",
+    allowMultiple: false,
+  },
+  {
+    name: "measures",
+    type: "column",
+    source: "source",
+    allowMultiple: true,
+  },
+]);
+
+function RenderBarChart(props) {
+  console.log(props);
+  return (
+    <ResponsiveContainer width="200%" height={500}>
+      <BarChart
+        data={props.data}
+        margin={{
+          top: 5,
+          right: 50,
+          left: 30,
+          bottom: 90,
+        }}
+        barCategoryGap={5}
+      >
+        <CartesianGrid strokeDasharray="3 3" />
+        <XAxis
+          dataKey={props.xAxisDataKey}
+          angle={-45}
+          textAnchor="end"
+          interval={0}
+        />
+        <YAxis />
+        <Tooltip />
+        <Legend verticalAlign="top" wrapperStyle={{ lineHeight: "40px" }} />
+        {props.measureNames.slice(0, 3).map((m, i) => ( // limiting number of bars rendered to max 3
+          <Bar
+            key={m}
+            dataKey={props.measureNames[i]}
+            fill={props.barFills[i]}
+          />
+        ))}
+      </BarChart>
+    </ResponsiveContainer>
+  );
+}
+
+function App() {
+  const config = useConfig();
+
+  const sigmaData = useElementData(config.source);
+  const columns = useElementColumns(config.source);
+
+  const dimension = sigmaData[config["dimension"]];
+  const measures = config.measures;
```

Comment:
this feels inconsistent, can we do `config.dimension` here? or combine into one line
```
const { dimension, measures } = config;
```